src/Dapr.Client/DaprClient.cs

```diff
@@ -783,6 +783,75 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             StateOptions stateOptions = default,
             IReadOnlyDictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+        /// <summary>
+        /// Define a SaveStateAsync task that gets a binary input   
+        /// </summary>
+        /// <param name="storeName">The  </param>
+        /// <param name="key"></param>
+        /// <param name="binaryValue"></param>
+        /// <param name="stateOptions"></param>
+        /// <param name="metadata"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public abstract Task SaveStateByteAsync(
+                       string storeName,
+                       string key,
+                       byte[] binaryValue,
+                       StateOptions stateOptions = default,
+                       IReadOnlyDictionary<string, string> metadata = default,
+                       CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Define a TrySaveStateAsync task that gets a binary input and gives a boolean output
+        /// </summary>
+        /// <param name="storeName"></param>
+        /// <param name="key"></param>
+        /// <param name="binaryValue"></param>
+        /// <param name="etag"></param>
+        /// <param name="stateOptions"></param>
+        /// <param name="metadata"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public abstract Task<bool> TrySaveStateByteAsync(
+                string storeName,
+                string key,
+                byte[] binaryValue,
```

Comment:
Same.

Reply:
I have changed SaveStateByteAsync, TrySaveStateByteAsync to use ReadOnlyMemory<byte>. Thanks for your review.